src/main/java/hw_8/Main.java

```diff
@@ -0,0 +1,28 @@
+package hw_8;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) {
+        MyBrowser myBrowser = new MyBrowser();
+        MyFile myFile = new MyFile();
+
+        Lecture30.finishWork(myBrowser);
+        Lecture30.finishWork(myFile);
+
+        @FunctionalInterface
+        interface SessionCloser {
+            void closeSession(List<CanBeClosed> entities);
+        }
```

Comment:
This functional interface should be ia a separate file. Please follow the approach from our lection

Reply:
done